guides/source-maps-many-domains.md

```diff
@@ -0,0 +1,21 @@
+If you'd like to use source maps with the same code that is deployed on many domains, use the following code (changing `yourdomainhere` to your domain in the `locRegex` variable):
+
+```javascript
+var _rollbarConfig = {
+  // ...
+  transform: function(payload) {
+    var trace = payload.data.body.trace;
+    var locRegex = /^(https?):\/\/[a-zA-Z0-9.-_]+\.yourdomainhere\.com(.*)/;
```

Comment:
This regular expression is almost surely not what you intended; `.-_` in the character class will actually match from ASCII 46 `.` to ASCII 95 `_`. The last three characters should be `._-` which means the literal characters `.`, `_` and `-` because `-` last means the literal hyphen instead of being used as a range.